### ISSUE TITLE
Increase the width of the post and bounty modals

### ISSUE THREAD

**Issue by humansinstitute** [stakwork/sphinx-tribes-frontend#775]
- Currently the modal for editing and creating new boutnies is fixed width and can be hard to read / edit for code heavy or longer bounties.
- The current width should be a minimum width it displays at.
- The width should be able to expand up to 75% of the screen size in which the modal is displayed.
- The total max width should be 2.5x the current minimum size.

![image](https://github.com/user-attachments/assets/da14326a-74b2-4e1f-b43d-ac1ef826a5d0)

## Acceptance Criteria

- [ ] Minimum width and mobile view on smaller windows is retained
- [ ] Maximum width of 250% of the fixed minimum value when viewing on large wide screens
- [ ] Width otherwise expands at 75% the total size of the window
- [ ] Both the edit and new bounties flows are updated at this new correct size

**Comment by Vayras** [comment]
Assign

**Comment by MahtabBukhari** [comment]
@humansinstitute please assign

**Comment by jordan-ae** [comment]
I'm free I can work on this

**Comment by humansinstitute** [comment]
@jordan-ae is having connection issues right now so can't confirm tehre - but bounty assigned: https://community.sphinx.chat/bounty/2985

**Comment by jordan-ae** [comment]
@humansinstitute I've fixed the width problem on the modal. The issue I'm noticing now is that all the components of the flow were built around the notion that the modal will be fixed in with. What do I do?

**Comment by humansinstitute** [comment]
The main thing that I am keen to expand into this space is the "Bounty Description"

Given there is a grid pattern where multiple components are in the same "column" as the description, could these be expanded to fill the space? 

E.g. Title would be wide, but select language etc smaller. 

Screenshot examples to follow

**Comment by humansinstitute** [comment]
![image](https://github.com/user-attachments/assets/180abdf0-954f-44aa-9abe-b0c160314a62)

**Comment by humansinstitute** [comment]
![image](https://github.com/user-attachments/assets/81f1bdf4-c2b8-4964-9723-dfde10fe8dad)

@jordan-ae - does this work for you? I don't mind if it doesn't look perfect as long as it's functional, most people don't see these screens.

**Comment by jordan-ae** [comment]
![Screenshot from 2024-12-16 12-32-07](https://github.com/user-attachments/assets/9e46b23f-55ff-49fc-9dee-c1bcc3c486c2)

With the specs you gave respected the modal kind of looks something like this. The components of the modal were also hardcoded to fix. A lot of hardcoding went into the whole process 😅. I think fixing the components should be part of the scope of this issue.

`minWidth: '712px',  // Retains the current minimum width`
`maxWidth: 'min(75vw, calc(712px * 2.5))',  // 2.5x the minimum width (712 * 2.5 = 1780)`
` width: '75vw', // Expands to 75% of the viewport width`
`height: '560px', // Retains the fixed height`
`margin: '0 auto'  // Ensures modal is centered horizontally`

**Comment by humansinstitute** [comment]
Agreed the components will need to expand into the space. I hadn't realised each was hardcoded. 

Does your PR update the component hardcoding?

**Comment by jordan-ae** [comment]
Yes the PR updates the components too

**Comment by humansinstitute** [comment]
@jordan-ae the update was supposed to affect the Edit modal? 

this is wider, but the components aren't increasing to fill the size? I has assumed this would be inherited from the create bounty form (that works), but looks like it's seperate. 

Can you update the PR?

**Comment by jordan-ae** [comment]
@humansinstitute I've opened a opened a PR fixing this
![Screenshot from 2024-12-17 11-01-36](https://github.com/user-attachments/assets/13d63340-0d83-47e4-8f7f-0fa57c79de72)
